### ISSUE TITLE
Slice causes overflow if BitMap contains 2**32-1

### ISSUE THREAD

**Issue by urdvr** [Ezibenroc/PyRoaringBitMap#41]
Example that demonstrates the issue:
```python
from pyroaring import BitMap
b = BitMap(range(10))
b.add(2**32-1)
b[10:] #OverflowError: value too large to convert to uint32_t
b[10:11] #OverflowError: value too large to convert to uint32_t
b[:] #OverflowError: value too large to convert to uint32_t
```

**Comment by urdvr** [comment]
Another (related?) overflow is even simpler to demonstrate.
```python
BitMap([2**32-1]) #OverflowError
```
while 
```python
b = BitMap()
b.add(2**32-1)
```
is fine.

**Comment by Ezibenroc** [comment]
I can reproduce your first example, all those slices cause an `OverflowError`. I will look into this.

Your second example is fine on my installation, this is weird.

**Comment by urdvr** [comment]
> Your second example is fine on my installation, this is weird.

Sorry, I doublechecked and I was wrong about the second example. 
I accidentally tested it in a dirty local branch that caused the slicing behavior to trigger. So the second example is invalid.

**Comment by Ezibenroc** [comment]
Fixed by 2715b82e788e097f4b9ca7aa1e8a814514b203d8, thank you for reporting the bug.
The `roaring.h` header has been updated a few months ago (https://github.com/RoaringBitmap/CRoaring/commit/820ed525b297d5bc2f288be0b8c9dc595cdabb33), but I did not apply this change on pyroaring side.
